Update multizone.rst (#770)
ansys/pyprimemesh: doc/source/user_guide/multizone.rst

> *This is a Beta feature. API Behavior and Implementation  may change in future.*

Reviewer: ```suggestion
*This is a beta feature. API behavior and implementation might change in the future.*
```

The :class:`MultiZoneControl <ansys.meshing.prime.MultiZoneControl>` class provides automatic geometry decomposition into mapped (sweepable) regions and free regions. 
Mapped (sweepable) regions are filled with hexahedral elements and free regions are filled with non-hexahedral elements.

> When you perform MultiZone Method meshing, all regions are meshed with a pure hexahedral mesh if possible.  

Author: @PipKat We are Write MultiZone like these in the documents, So as of now I'm keeping the word as MultiZone and resolving these conversation

> The :class:`AutoMesh <ansys.meshing.prime.automesh>` class enables you to automatically create the hex mesh on the scoped bodies using multizone meshing algorithms. 

Reviewer: ```suggestion
The :class:`AutoMesh <ansys.meshing.prime.automesh>` class lets
 you automatically create the hex mesh on the scoped bodies using multizone meshing algorithms. 
```

> :func:`AutoMesh.mesh() <ansys.meshing.prime.AutoMesh.mesh>` method allows you to perform MultiZone meshing with given MultiZone control. 

Reviewer: ```suggestion
The :func:`AutoMesh.mesh() <ansys.meshing.prime.AutoMesh.mesh>` function lets you perform multizone meshing with the given multizone control. 
```

> The below example shows how MultiZone control can be applied on a body: 

Reviewer: ```suggestion
The following example shows to apply multizone controls on a body: 
```

> 1. Start the PyPrimeMesh client and read the model. The model is made up of two topo volumes that share a connected topo face between them. The two topo volume have volume zones defined such as left volume zone is **solid1** and right is **solid**.

Reviewer: ```suggestion
1. Start the PyPrimeMesh client and read the model. The model is made up of two topo volumes that share a connected topo face between them. The two topo volumes have volume zones defined. The left volume zone is ``solid1``, and the right volume zone is ``solid``.
```
Use double backticks for the names as they are code entities.

   multizone_control = model.control_data.create_multi_zone_control()

> 3. Define the volume scope and surface scope within the model and applying the volume scope and surface scope to the Multizone Control. In this example, volume scope is scoped specifically to "solid1" to show the difference between the MultiZone mesh and automesh

Reviewer: ```suggestion
3. Define the volume scope and surface scope within the model and apply the volume scope and surface scope to the multizone control.
   In this example, the volume scope is scoped specifically to ``solid1`` to show the difference between the multizone mesh and automesh.
```

.. note::

>   Keep the surface scope as the complete geometry (*)

Reviewer: ```suggestion
  Keep the surface scope as the complete geometry (*).
```I don't understand the significance of the "(*)" here. Perhaps additional clarification is needed?

Author: @PipKat For complete geometry we are adding it as *
